doc/api/v0.yaml

```diff
@@ -559,3 +574,16 @@ components:
           items:
               type: integer
               format: i64
+
+    VoterGroup:
+      properties:
+        id: integer
+          description: Unique identifier of this voter group
+        name: string
+          description: Voter group name
+        voting_token:
+          type: string
+          format: [0-9a-f]{56}(\.[0-9a-f]{1,64})?
+          description: |
+            The identifier of voting power token used withing this group. All vote plans within a
+            group are guaranteed to use the same token.
```

Comment:
> All vote plans within a group are guaranteed to use the same token

Why does this matter?

Reply:
To simplify the actual UX.

1. You choose the group to use.
2. Within this group you have voting power `x`.
3. You see the proposals that you can vote for with this voting power.

Reply:
Thinking of voting group as roles, I might be in the technical council and within that group participate in different elections (e.g. inter-technical team, everybody same voting power, and with the whole council, where maybe different people will be required to have different powers). IMHO it would be inconvenient for me as a user to select the right voting group each time when clearly they are related to the same role.

Reply:
Btw, don't think we define vote plans in a voting group anyway so I don't know how to interpret the last sentence

Reply:
> Btw, don't think we define vote plans in a voting group anyway so I don't know how to interpret the last sentence

Aren't those implicit?

Reply:
I think they are implicitly defined by looking at what voteplan use the same token as the one of voting group X, which makes the statement a bit weird (i.e. could not be otherwise defined what are the vote plans in a voting groups)